torba/torba/client/basenetwork.py

```diff
@@ -136,6 +139,10 @@ def rpc(self, list_or_method, args):
         await session.send_request('server.banner')
         return session
 
+    def _update_remote_height(self, header_args):
+        if header_args and header_args[0]:
+            self.remote_height = header_args[0]["height"]
```

Comment:
Does this get called with invalid header_args? Just wondering what case you'd need the `if header...` check?